setup.py

```diff
@@ -25,7 +25,7 @@ def package_data(pkg, root_list):
 
 setup(
     name='xblock-group-project-v2',
-    version='0.11.0',
+    version='0.11.1',
```

Comment:
This includes pretty big changes so I suggest we bump the minor version. 
```suggestion
    version='0.12.0',
```